commands/create_cluster.go

```diff
@@ -30,14 +30,16 @@ type addClusterArguments struct {
 	inputYAMLFile           string
 	numWorkers              int
 	owner                   string
+	releaseVersion          string
 	scheme                  string
 	token                   string
 	wokerAwsEc2InstanceType string
 	wokerAzureVMSize        string
 	workerNumCPUs           int
 	workerMemorySizeGB      float32
 	workerStorageSizeGB     float32
-	releaseVersion          string
+	workersMax              int64
```

Comment:
Why int64 instead of just `int` ? I assume this has to do with the generated client?

Reply:
The generated client works with `int64` and I preferred to work with one data type instead of doing conversions somewhere.